app/assets/stylesheets/application.css.scss

```diff
@@ -35,4 +35,16 @@
 
 .block_user_current_block_id {
   margin-left: 20px;
+}
+
+#flickr_image a {
+  margin-bottom: 15px;
+}
+
+#flickr_images {
```

Comment:
Avoid using id selectors<br>Selector `flickr_images` should be written in lowercase with hyphens